manifest-gen/src/main.rs

```diff
@@ -42,6 +43,15 @@ pub struct Args {
     /// Trustee namespace where to install trustee configuration
     #[arg(long, default_value = "operators")]
     trustee_namespace: String,
+
+    #[arg(
+        long,
+        default_value = "quay.io/confidential-clusters/compute-pcrs:latest"
+    )]
+    pcrs_compute_image: String,
+
+    #[arg(long, default_value = "quay.io/fedora/fedora-coreos:42.20250705.3.0")]
```

Comment:
I would remove this and hardcoded directly in the compute-pcrs. We will then add a logic to detect the coreos version to calculate

Reply:
Yes, but the job with the image volume is created by the operator, so you don't need this in the manifests